api/config/services.yaml

```
@@ -103,6 +103,12 @@ services:
     App\OpenApi\OAuthDecorator:
         decorates: 'api_platform.openapi.factory'
 
+    App\OpenApi\RefreshTokenDecorator:
+      decorates: 'api_platform.openapi.factory'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)